### ISSUE TITLE
[Bug] TypeError: Cannot read properties of undefined (reading 'CLAMP_TO_EDGE')

### ISSUE THREAD

**Issue by tungvip2143** [OHIF/Viewers#4344]
### Describe the Bug

The error occurs when I run the build locally using the command `npx serve ./dist -c ../public/serve.json`.
![Ảnh màn hình 2024-08-22 lúc 11 31 29](https://github.com/user-attachments/assets/eab7d630-b604-4a79-b519-0616c93cc6b3)

### Steps to Reproduce

I followed the steps from the [OHIF Documentation - Build for Production](https://docs.ohif.org/deployment/build-for-production/),

### The current behavior

IThe build does not run

### The expected behavior

The build runs successfully.

### OS

macos 14.5

### Node version

18.16.1

### Browser

Chrome 127.0.6533.120

**Comment by sedghi** [comment]
probably you are not using the latest OHIF

**Comment by ahlaughland** [comment]
I'm getting the same error with OHIF  `"version": "3.9.0-beta.87",`  via a github workflow. I can't track it to a single version of OHIF because `3.9.0-beta.86` gives the same error when running a build. I also downgraded ` "@cornerstonejs/core": "^1.82.4",` to ` "@cornerstonejs/core": "^1.77.6",` but I get the same error.

**Comment by ahlaughland** [comment]
After digging deeper I think the new external dependency `@externals/dicom-microscopy-viewer` is not being bundled by webpack which is how it should work. But the error is in the webpacked external dependency for `dicom-microscopy-viewer` in `platform\app\dist\dicom-microscopy-viewer.bundle.9ff1c6609fa304dd8010.js`

**Comment by sedghi** [comment]
Can you check the latest OHIF viewer, which now includes Cornerstone 3D 2.0?

Here are the migration guides:

Try OHIF locally: [https://viewer-dev.ohif.org/localbasic](https://viewer-dev.ohif.org/localbasic)<br>Try Cornerstone3D 2.0 locally: [https://www.cornerstonejs.org/live-examples/local.html](https://www.cornerstonejs.org/live-examples/local.html)

[https://docs.ohif.org/migration-guide/3p8-to-3p9/](https://docs.ohif.org/migration-guide/3p8-to-3p9/)

**Comment by ahlaughland** [comment]
Here are some examples of:
_Try OHIF locally: https://viewer-dev.ohif.org/localbasic_

I loaded CTs and tested all the toolbar options everything seems to work - no obvious problems. 3D is slow loading on my laptop the specs are below. 

I loaded NM images but get an error:
![image](https://github.com/user-attachments/assets/cb868549-5666-448f-99f7-852b8c104a5a)

Clicking `Show Details` doesn't show anything.

Here are some examples of:
_Try Cornerstone3D 2.0 locally: https://www.cornerstonejs.org/live-examples/local.html_

Modality CT:
![image](https://github.com/user-attachments/assets/e03bd6ea-4612-474c-a357-abb7c2313a20)

Modality NM:
![image](https://github.com/user-attachments/assets/30e76f22-1f3a-44c5-9256-2bd67ab37192)


Laptop (test machine) specs:
![image](https://github.com/user-attachments/assets/083b14ec-0000-42e4-ad14-1a339b2e48fc)


I reviewed the migration guide looks good, but won't be migrating to v2 until Q1 2025 so I have no practical experience yet:
https://docs.ohif.org/migration-guide/3p8-to-3p9/

**Comment by sedghi** [comment]
@ahlaughland If this CLAMP issue is not happening anymore, maybe we close it and open another issue for NM?

**Comment by ahlaughland** [comment]
@sedghi I agree, we can close this. I haven't any problems since I upgraded back in September and I think it was related to my merge process anyway.
